vnext/template/metro.devMode.config.js

```diff
@@ -1,2 +1,49 @@
-const {makeMetroConfig} = require('@rnw-scripts/metro-dev-config');
-module.exports = makeMetroConfig();
```

Comment:
one option would be to still use this code, but then apply whatever deltas are needed to make it work in dev mode; that way if we make changes to the main metro config, this won't get out of sync